main.go

```diff
@@ -104,6 +104,36 @@ func main() {
 		logg.Fatal(err.Error())
 	}
 
+	//get max asset sizes
+	cfg := core.Config{
+		MaxAssetSize: make(map[db.AssetType]*uint64),
+	}
+	maxAssetSizes := strings.Split(mustGetenv("CASTELLUM_MAX_ASSET_SIZES"), ",")
+	for _, v := range maxAssetSizes {
+		sL := strings.Split(v, "=")
+		if len(sL) != 2 {
+			logg.Fatal("expected a max asset size configuration value in the format: '<asset-type>:<max-asset-size>', got: %s", v)
+		}
+		assetType := sL[0]
+		maxSize, err := strconv.ParseUint(sL[1], 10, 64)
+		if err != nil {
+			logg.Fatal(err.Error())
+		}
+
+		found := false
+		for _, assetManager := range team {
+			for _, info := range assetManager.AssetTypes() {
+				if assetType == string(info.AssetType) {
+					found = true
+					cfg.MaxAssetSize[info.AssetType] = &maxSize
+				}
+			}
+		}
```

Comment:
This might be worth extracting into an `IsAssetType` method on type AssetManager.

Reply:
Since this is the only place where we use this particular piece of code, we can leave it as is. If we need it somewhere else in the future then we can extract it into a re-usable unit.